### ISSUE TITLE
CORS is not working when default version fires for POST request

### ISSUE THREAD

**Issue by calebbriancescoGL** [p-meier/hapi-api-version#15]
I have set the path "/v1/images/thumbnails" but it wont work if I send the post request to "/images/thumbnails", first I get a 404 over the method and then I get the following error: 
```
XMLHttpRequest cannot load http://api.photoandgo.local/images/thumbnail. Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://bocaraton.local.photoandgo.com:5002' is therefore not allowed access. The response had HTTP status code 404.
```

**Comment by p-meier** [comment]
Please check if #17 fixes your issue. I am building a new version today...

**Comment by timcosta** [comment]
Closing this issue as fixed due to lack of response. I was experiencing the same issue on 1.3.0 and can confirm that it was fixed in 1.4.0.
